### PR TITLE
[9.19] Added user events

### PR DESCRIPTION
**Pull request by jvandervalk**
I've added a missing API call to /users/:id/events to retrieve events for a specified user
See https://docs.gitlab.com/ee/api/events.html#get-user-contribution-events

### CODE REVIEW

File: lib/Gitlab/Api/Users.php [hunk 1]
```diff
@@ -448,4 +448,44 @@ public function removeImpersonationToken($user_id, $impersonation_token_id)
     {
         return $this->delete('users/'.$this->encodePath($user_id).'/impersonation_tokens/'.$this->encodePath($impersonation_token_id));
     }
+
+    /**
+     * @param int|string $user_id
```
Comment (GrahamCampbell) [inline lib/Gitlab/Api/Users.php]: when is the user id a string?

**Review by GrahamCampbell — COMMENTED** [review]
Thanks for this. I just have one minor comment. :)

**Comment by GrahamCampbell** [comment]
Excellent, thanks. 🍻

You can use this code by installing `m4tthumphrey/php-gitlab-api:^9.19@dev` until v9.19.0 is tagged. The current plan is to release v9.19.0 and v10.0.0 simultaneously in August.

File: lib/Gitlab/Api/Users.php (hunk 1 above)
Comment (jvandervalk) [reply lib/Gitlab/Api/Users.php]: Thought maybe if someone saved it in his own database, but now I have a closer look at User.php I don't see it elsewhere, I'll change it!

**Pull request merged**